content/manuals/scout/policy/configure.md

```diff
@@ -1,52 +1,68 @@
+Once you've edited the policy parameters, the customized version of the policy
+overrides the default configuration.
+
+You can edit the display name and description of the new policy to help
+better communicate the compliant and non-compliant states of the policy.
+You can not change the name of the policy type, only its display names.
 
-The available configuration parameters for a policy depends on the default
-policy you used as a base for your custom policy.
+The available configuration parameters for a policy depends on the
+policy type that you're editing. For more information, refer to
+[Policy types](/manuals/scout/policy/_index.md#policy-types).
 
 To configure a policy:
 
 1. Go to the [Policies page](https://scout.docker.com/reports/policy) in the Docker Scout Dashboard.
-2. Select the policy you want to configure.
-3. Select **View policy details** to open the policy side panel.
+2. Select the **Add policy** button to open the policy configuration screen.
+3. On the policy configuration screen, locate the policy type that you want to
+   configure, and select **Configure** to open the policy configuration panel.
+
+   - If the **Configure** button is grayed out, it means the selected policy
+     has no configurable parameters.
+   - If the button reads **Integrate**, it indicates that setup is required
+     before the policy can be enabled. Selecting **Integrate** will direct you
+     to the policy's setup guide.
 
    If this button is grayed out, then the selected policy doesn't have any
    configuration parameters.
 
-4. In the side panel, select **Copy to customize** to open the policy configuration page.
-5. Update the policy parameters.
-6. Save the changes:
+   If the button says **Integrate**, it's an indication that you need to
+   complete a setup before the policy can be enabled. Selecting the
+   **Integrate** button takes you to the setup guide for that policy.
+
+4. Update the policy parameters.
+5. Save the changes:
 
    - Select **Save and enable** to commit the changes and enable the policy for
      your current organization.
    - Select **Save policy** to save the policy configuration without enabling
      it.
 
-## Disable a policy
+## Delete a policy
 
-When you disable a policy, evaluation results for that policy are hidden, and
-no longer appear in the Docker Scout Dashboard or in the CLI. Historic
-evaluation results aren't deleted if you disable a policy, so if you change
-your mind and re-enable a policy later, results from earlier evaluations will
-still be available.
+When you delete a policy, evaluation results for that policy are hidden, and
+no longer appear in the Docker Scout Dashboard or in the CLI.
 
-To disable a policy:
+To delete a policy:
 
 1. Go to the [Policies page](https://scout.docker.com/reports/policy) in the Docker Scout Dashboard.
 2. Select the policy you want to disable.
-3. Select the **Disable** button.
+3. Select the **Delete** button.
```

Comment:
**suggestion:** The Delete button for now is in the image details side panel so this should be step 4. Step 3 is:
```
3. Click on "View policy details"
```
<img width="371" alt="image" src="https://github.com/user-attachments/assets/13953385-7cf5-4483-af75-1eab439923ba">